app/views/schools/participants/_currently_training_sorted_by_mentor.html.erb

```diff
@@ -0,0 +1,69 @@
+<p class="govuk-body">Sort by:
+  <b class="govuk-!-margin-left-2">Mentor (A-Z)</b>
+  <%= govuk_link_to("Induction start date",
+                    school_participants_path(school, sorted_by: :induction_start_date),
+                    no_visited_state: true,
+                    class: ["govuk-!-margin-left-2"]) %>
+</p>
```

Comment:
Rather than duplicating this across both partials (which does work!) would it be better to extract it into its own mini component, and then tell that component which sort state is currently active? Just an idea!

Reply:
Good idea! That's what components are for!.